website/app.vue

```diff
@@ -38,53 +33,73 @@ const localize = lang => {
 }
 localize(lang)
 
-export default {
+export default defineComponent({
   name: 'App',
 
-  computed: {
-    lang() {
-      return this.$route.path.split('/')[1] || Language.CN
-    },
-    isComponent() {
-      return /^component-/.test(this.$route.name || '')
-    },
-  },
+  setup(){
+    const route = useRoute()
 
-  watch: {
-    lang(val) {
-      if (val === Language.CN) {
-        this.suggestJump()
-      }
-      localize(val)
-    },
-  },
+    const lang = computed(() => route.path.split('/')[1] || Language.CN)
 
-  mounted() {
-    localize(this.lang)
-    if (this.lang === Language.CN) {
-      this.suggestJump()
-    }
-  },
+    const isComponent = computed(() => /^component-/.test(route.name || ''))
 
-  methods: {
-    suggestJump() {
+    const _this = getCurrentInstance()
+
+    const suggestJump = () => {
       if (process.env.NODE_ENV !== 'production') return
 
       const href = location.href
       const preferGithub = localStorage.getItem('PREFER_GITHUB')
       const cnHref = href.indexOf('element-plus.gitee.io') > -1
       if (cnHref || preferGithub) return
       setTimeout(() => {
-        if (this.lang !== Language.CN) return
-        this.$confirm('建议大陆用户访问部署在国内的站点，是否跳转？', '提示')
+        if (lang.value !== Language.CN) return
+        _this.appContext.config.globalProperties.$confirm('建议大陆用户访问部署在国内的站点，是否跳转？', '提示')
           .then(() => {
             location.replace('https://element-plus.gitee.io')
           })
           .catch(() => {
             localStorage.setItem('PREFER_GITHUB', 'true')
           })
       }, 1000)
-    },
+    }
+
+    watch(() => lang.value, val => {
+      if (val === Language.CN) suggestJump()
+      localize(val)
+    })
+
+    onMounted(() => {
+      localize(lang.value)
+
+      if (lang.value === Language.CN) suggestJump()
+    })
+
+    return {
+      lang,
+      isComponent,
+    }
   },
-}
+
+  render() {
+    const mainHeader = this.lang !== 'play' ? h(MainHeader, {
+      style: 'position: fixed;top: 0;width: 100%;z-index: 2000',
+    }) : null
+
+    const mainFooter = this.lang !== 'play' && !this.isComponent ? h(MainFooter) : null
+
+    const content = [h('div', {
+      class: 'main-cnt',
+    }, [h(RouterView)]), mainFooter]
+
+    const contentWrapper = !this.isComponent ? h(ElScrollbar, null, { default: () => content }): content
```

Comment:
`const notComponent = !isComponent` since you have used twice.
also either break branches into lines or add a space before the `colon`

Reply:
```suggestion
const contentWrapper = !this.isComponent 
  ? h(ElScrollbar, null, { default: () => content })
  : content
```
or
```suggestion
    const contentWrapper = !this.isComponent ? h(ElScrollbar, null, { default: () => content }) : content

```